textsubs.lic

```diff
@@ -784,19 +386,6 @@ TextSubs.add('decent (quantity|number) remains to be found', 'decent \1 (2/5) re
 TextSubs.add('small (quantity|number) remains to be found', 'small \1 (1/5) remains to be found')
 TextSubs.add('scattering of (resources|trees) remains to be found', 'scattering of \1 (0/5) remains to be found')
 
-TextSubs.add('Encumbrance : None', 'Encumbrance : None (0/11)')
-TextSubs.add('Encumbrance : Light Burden', 'Encumbrance : Light Burden (1/11)')
-TextSubs.add('Encumbrance : Somewhat Burdened', 'Encumbrance : Somewhat Burdened (2/11)')
-TextSubs.add('Encumbrance : Burdened', 'Encumbrance : Burdened (3/11)')
-TextSubs.add('Encumbrance : Heavy Burden', 'Encumbrance : Heavy Burden (4/11)')
-TextSubs.add('Encumbrance : Very Heavy Burden', 'Encumbrance : Very Heavy Burden (5/11)')
-TextSubs.add('Encumbrance : Overburdened', 'Encumbrance : Overburdened (6/11)')
-TextSubs.add('Encumbrance : Very Overburdened', 'Encumbrance : Very Overburdened (7/11)')
-TextSubs.add('Encumbrance : Extremely Overburdened', 'Encumbrance : Extremely Overburdened (8/11)')
-TextSubs.add('Encumbrance : Tottering Under Burden', 'Encumbrance : Tottering Under Burden (9/11)')
-TextSubs.add('Encumbrance : Are you even able to move?', 'Encumbrance : Are you even able to move? (10/11)')
-TextSubs.add('Encumbrance : It\'s amazing you aren\'t squashed!', 'Encumbrance : It\'s amazing you aren\'t squashed! (11/11)')
```

Comment:
Leave these unless I missed another Simu toggle you can remind me about Nituki. 😁

Reply:
Nope you are right. I thought these were part of the luck update but had another sub running when I checked. These should be kept.